### PR TITLE
updated markdown to html converter

### LINKED ISSUE #869: Markdown to HTML converter
# Description

This Script would convert markdown files into HTML. Can be extremely useful for developers who are implementing modern text enhancement features as most blog sites, note apps etc. are adapting markdown support.

## Language

- [X] Javascript
- [X] Python

## Checklist:

- **Details on the Script**

| name            | about                                                  | labels  | assignees                                                      |
| --------------- | ------------------------------------------------------ | ------- | -------------------------------------------------------------- |
| Markdown to HTML converter | Easily convert Markdown to HTML by use of available parsing libraries | Python, JavaScript, good-first-issue | self-assign if you want to work on this issue else leave blank |

**Describe the solution you'd like**: A clear and concise description of what you want to happen.

There are various PyPI libraries or npm packages available for this implementation. The idea is to add an automation script that can demonstrate how this can be done.

This has a lot of utility and thus this can be a good start point for anyone who wants to explore libraries. And of course, this is not limited to just conversion, but on top of it some good use-cases can be added as examples, that can be good.

This can be a good script to work on for beginners. Would work closely with the assignees for this implementation.

### PR DESCRIPTION
**Pull request by divyanshu887**
# Description

i have added the code for converting markdown to html.

Fixes #869 

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
- [x] Documentation Update

### CODE REVIEW

File: JavaScript/Markdown_to_Html/script.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
+var output = '';
```
Comment (sohamsshah) [inline JavaScript/Markdown_to_Html/script.js:1]: Avoid the usage of `var`. Alternatively use `let` and `const` as following the ES6 Syntax

File: JavaScript/Markdown_to_Html/script.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+inputTextarea = document.querySelector('#inputTextarea');
+outputTextarea = document.querySelector('#outputTextarea');
+buttonClear = document.querySelector('#buttonClear');
+buttonCopyHTML = document.querySelector('#buttonCopyHTML');
+buttonCopyMarkdown = document.querySelector('#buttonCopyMarkdown');
+buttonPreview = document.querySelector('#buttonPreview');
+
```
Comment (sohamsshah) [inline JavaScript/Markdown_to_Html/script.js:8]: As an enhancement, can you make UDFs to enable more structured code- based on the operations? In a nutshell, the idea is to remove all global vars.

**Review by sohamsshah — CHANGES_REQUESTED** [review]
First of all, thanks for this addition. Functionality-wise this seems great to me! 

I have suggested some small changes. After that, the PR is ready to merge from my side!

File: JavaScript/Markdown_to_Html/script.js (hunk 2 above)
Comment (divyanshu887) [reply JavaScript/Markdown_to_Html/script.js:2]: @sohamsshah  i have done all additions mentioned above.

**Review by sohamsshah — APPROVED** [review]
LGTM @HarshCasper

**Review by HarshCasper — CHANGES_REQUESTED** [review]
We don't want GUI Apps as part of the project. 

Thanks for contributing!

**Comment by sohamsshah** [comment]
@divyanshu887 You can transform this to a Node.js CLI app easily I suppose and that would be accepted. Feel free to raise a fresh PR for the same. Let me know if you want any help regarding implementation of this :)

**Pull request closed without merging**